12-connectivity.tf

```
@@ -68,6 +68,7 @@ resource "azurerm_subnet" "additional_subnets" {
   address_prefixes     = [each.value.address_prefix]
   resource_group_name  = var.resource_group_name
   virtual_network_name = azurerm_virtual_network.virtual_network.name
+  enforce_private_link_endpoint_network_policies = each.value.additional_subnets_priv_link_endpoint_policy
```

this means you don't have to specify it
```suggestion
  enforce_private_link_endpoint_network_policies = lookup(each.value, "enforce_private_link_endpoint_network_policies", true)
```

---

also we could probably just enable it by default?

```suggestion
  enforce_private_link_endpoint_network_policies = true
```

I can't see a reason we wouldn't have this on